server/src/main/java/com/broadcom/lsp/domain/cobol/event/model/RunAnalysisEvent.java

```diff
@@ -22,7 +22,15 @@
  * all the registered documents.
  */
 public class RunAnalysisEvent extends DataEvent {
+  public final boolean quiet;
+
   public RunAnalysisEvent() {
     super(RUN_ANALYSIS_EVENT, RUN_ANALYSIS_EVENT.getId());
+    this.quiet = true;
+  }
+
+  public RunAnalysisEvent(boolean quiet) {
+    super(RUN_ANALYSIS_EVENT, RUN_ANALYSIS_EVENT.getId());
+    this.quiet = quiet;
```

Comment:
I think that "verbose" here would be more descriptive, but it depends on how do you use it

Reply:
Agree, I'll rename it.